src/main/java/Utils/StringUtils.java

```diff
@@ -1,10 +1,25 @@
 package Utils;
 
+import java.util.List;
+
 public class StringUtils {
 
     public static boolean isEmptyTrimmed(String str) {
         return (str.trim().isEmpty());
     }
 
+    public static boolean isFullyContainedInArray(List<String> containedArray, List<String> containingArray) {
```

Comment:
You can make it easier.
Instead of this function, create this one:
`public static List<String> listToUpperCase(List<String> lst) {
        return lst.stream().map(String::toUpperCase).collect(Collectors.toList());
    }`

and later use the list containsAll method:
`listToUpperCase(haystack).containsAll(listToUpperCase(needle));`

Reply:
Done.